src/sentry/api/helpers/events.py

```diff
@@ -52,17 +52,33 @@ def get_direct_hit_response(
 
 
 def get_query_builder_for_group(
-    query: str, snuba_params: ParamsType, group: Group, limit: int, offset: int
+    query: str,
+    snuba_params: ParamsType,
+    group: Group,
+    limit: int,
+    offset: int,
+    orderby: str | None = None,
 ) -> QueryBuilder:
     dataset = Dataset.IssuePlatform
     if group.issue_category == GroupCategory.ERROR:
         dataset = Dataset.Events
+    selected_columns = ["id", "project.id", "issue.id", "timestamp"]
+
+    if orderby is None:
+        orderby = "-timestamp"
+    elif orderby == "sample":
```

Comment:
is it possible to order by `-sample`?

Reply:
it wouldn't really make any sense, because it would basically mean _"pseudo-random order but descending"._ 

That being said, on the calling side there are only two values used, either `"sample"` or `None` to use the default.